CGAL_ImageIO/src/CGAL_ImageIO/gis_impl.h

```diff
@@ -143,7 +143,7 @@ int writeGis( char *name, _image* im) {
 		if ( j<n && i<size ) sprintf( str+strlen(str), " " );
 	      }
 	      sprintf( str+strlen(str), "\n" );
-	      res = ImageIO_write( im, str, strlen( str )  );
+	      res = static_cast<int>(ImageIO_write( im, str, strlen( str )));
 	      if ( res  <= 0 ) {
```

Comment:
Using `size_t` is not possible because `res` is also used before to get the result of `writeGisHeader`. I suggest keeping my `static_cast` and just fix the `if`. Another option is to create a temporary size_t variable, but I'm not sure it's worth it.